### ISSUE TITLE
Document how to use dialogs

### ISSUE THREAD

**Issue by thusc** [smartcoop/design#141]
- Dialogs are documented at https://design.smart.coop/development/docs/c-dialog.html
- The documentation should mention the full example page https://design.smart.coop/development/template-examples/dialog.html
- It should mention that in addition of the code displayed on that page, an additional `<div class="c-dialog-context c-dialog-context--visible">` wrapper is needed
- It should mention that such code can be appended after the main `div` of the page (the one that usually contains the `header` and the `main`), just before the `script` things.

**Comment by thusc** [comment]
Nice, that's a great start. Something else to add: how to start the page with the dialog initially hidden.

**Comment by Wolfr** [comment]
I think the dialogs are hidden by default, no?

**Comment by thusc** [comment]
Yes they are but since I copied from the example templates, it was not obvious for me at first sight what I should have changed. Then I looked within the CSS if there was something else beside `--visible`, and then tried without that. I think some prose at the top of the documentation page could help a bit.

**Comment by Wolfr** [comment]
In your initial comment, you write that “It should mention that in addition of the code displayed on that page, an additional <div class="c-dialog-context c-dialog-context--visible"> wrapper is needed”

It's true that this is a wrapper element, but it's used for every singe modal. I added some better examples here: https://design.smart.coop/development/template-examples/dialog.html

I updat the docs at the top: https://design.smart.coop/development/docs/c-dialog.html
